lib/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { Environment } from "@standardnotes/snjs";
+
+declare global {
+  interface Window { msCrypto: unknown; }
+}
+
+export const generateUuid = () => {
```

Comment:
why not use `uuid` package ? https://github.com/standardnotes/syncing-server-js/blob/72184b0f7619ab316a677c4435b2c1cfde51c5b3/src/Domain/Session/SessionService.ts#L31

Reply:
Only if the payload size of uuid package is not unnecessarily exorbitant.